scripts/build_podspecs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -eu
+
+function usage() {
+  echo "$0 [-u] VERSION"
+  echo
+  echo "OPTIONS:"
+  echo "  -u: Additionally, upload the podspecs as they are generated"
+}
+
+upload=false
+while getopts ":u" opt; do
+  case $opt in
+    u)
+      upload=true
+      ;;
+    \?)
+      usage
+      exit 1
+      ;;
+  esac
+done
+shift "$((OPTIND-1))"
+
+if [[ $# -eq 0 ]]; then
+  echo "Must provide target version"
+  exit 1
+fi
+
+VERSION=$1
+NEWLINE=$'\n'
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+tmpfile=$(mktemp -d /tmp/.build_podspecsXXXXXX)
+echo "Building podspecs in $tmpfile"
+
+# We have a name transformation here. We want the pod names to be
+# SwiftX or CX, but our target names aren't. We add SwiftX to the front
+# of all NIO targets, and then remove it in the source files declaration
+# if needed.
+TARGETS=$(${here}/list_topsorted_dependencies.sh -l -r | grep -v "NIOPriorityQueue" | sed 's/^NIO/SwiftNIO/')
+
+for target in $TARGETS; do
+  echo "Building podspec for $target"
+
+  # We need to suppress the return code of grep here.
+  RAW_DEPENDENCIES=$(${here}/list_topsorted_dependencies.sh -d "${target#Swift}" | grep -v "NIOPriorityQueue" | sed 's/^NIO/SwiftNIO/')
+  DEPENDENCIES=""
+
+  for raw_dependency in $RAW_DEPENDENCIES; do
+    DEPENDENCIES="${DEPENDENCIES}${NEWLINE}  s.dependency '$raw_dependency', s.version.to_s"
```

Comment:
this should be array append

    dependencies+=( "${dependencies}${newline}  s.dependency '$raw_dependency', s.version.to_s" )